### ISSUE TITLE
Split Editor

### ISSUE THREAD

**Issue by markub3327** [arduino/arduino-ide#909]
Hello,

  I need to split the editor into twice windows. The same feature is in VS Code by pressing Split Editor Right (COMMAND+\ on macOS).

Thanks a lot.
Have a nice day.

**Comment by kittaakos** [comment]
Possible related: https://github.com/arduino/arduino-ide/issues/552

You can do <kbd>F1</kbd> > type _Split Editors_ > and press <kbd>Enter</kbd> but then you cannot close the split tabs.

**Comment by CptHolzschnauz** [comment]
Let's rephrase it to "we would like to have" 

Thank you contributors for all the work done!

PS: RC5 : Still two main issues IMHO: Split view and serial monitor

**Comment by markub3327** [comment]
Thanks a lot. I wait to new releases.

**Comment by CaptainCredible** [comment]
Really hoping this will be be implemented <3
